config/wikis/trwiki.yaml

```diff
@@ -30,17 +32,24 @@ autolabeled_samples:
 merged_samples:
   labeled_revisions.20k_2015:
     - human_labeled_revisions.20k_2015
+  labeled_revisions.3k_2020:
+    - human_labeled_revisions.3k_2020
```

Comment:
You'll want to merge this with `autolabeled_revisions.20k_2020`

So this block should look like this: 
```
merged_samples:
  labeled_revisions.20k_2015:
    - human_labeled_revisions.20k_2015
  labeled_revisions.20k_2020:
    - human_labeled_revisions.3k_2020
    - autolabeled_revisions.20k_2020
```

Reply:
Thank you Aaron, this has been fixed.